sources/web/datalab/templates/nb.html

```diff
@@ -161,6 +161,15 @@
                 <li id="interruptKernelButton"><a href="#">Interrupt Execution</a></li>
               </ul>
             </div>
+            <div class="btn-group">
```

Comment:
Adding another button to the toolbar will likely need changes to the minimum width of screen, and the media size decorators.